CreateService/handler.ts

```diff
@@ -147,6 +148,21 @@ const createServiceTask = (
     200
   );
 
+export const getAuthorizedRecipientsFromPayload = (
+  servicePayload: ServicePayload
+): O.Option<null | ReadonlyArray<FiscalCode>> =>
```

Comment:
Why `null`? `Option`s are used to avoid null-ish values, if we may encounter a `null` maybe we are doing it wrong.

Reply:
Fix with cecbe9f